plugins/twitter/requirements.txt

```diff
@@ -0,0 +1,14 @@
+black
+isort
+flake8
+pylint
+abstract-singleton
+wheel
+setuptools
+build
+twine
+tweepy
+pandas
+auto_gpt_plugin_template
+python-dotenv
+pathlib
```

Comment:
Didn't really consider this. Make a new file in the base of the repo called requirements_all.txt with the following contents:

```
# Auto-GPT Plugins Full dependency set
-r requirements.txt

# plugins/twitter
tweepy
```

And include any specific packages you need